src/OneExplorer/OneExplorer.ts

```diff
@@ -174,29 +174,30 @@ export class OneTreeDataProvider implements vscode.TreeDataProvider<OneNode> {
     let warningMessage;
     if (oneNode.node.type === NodeType.baseModel) {
       // TODO automatically change the corresponding files
-      warningMessage = 'WARNING: You may need to change input paths in the following cfg files:\n';
-      for (const conf of oneNode.node.childNodes) {
-        // NOTE A newline character is not supported in input box.
-        // vscode-extension doesn't plan to support multiple lines inside in input box or in
-        // notification box.
-        warningMessage += `${conf.name} `;
-      }
-    } else {
-      warningMessage = 'WARNING: Renaming may result in some unexpected changes on the tree view.';
+      warningMessage = `WARNING! ${
+          oneNode.node.childNodes.map(node => `'${node.name}'`)
+              .toString()} will disappear from the view.`;
+    } else if (oneNode.node.type === NodeType.derivedModel) {
+      // TODO automatically change the corresponding files
+      warningMessage = `WARNING! '${oneNode.node.name}' may disappear from the view.`;
     }
 
     const validateInputPath = (newname: string): string|undefined => {
       const oldpath = oneNode.node.path;
       const dirpath = path.dirname(oneNode.node.uri.fsPath);
       const newpath: string = path.join(dirpath, newname);
 
-      if (path.extname(newpath) !== path.extname(oldpath)) {
-        return `File ext must be (${path.extname(oldpath)})`;
+      if (!newname.includes(path.extname(oldpath))) {
```

Comment:
```suggestion
      if (!newname.endsWith('.' + path.extname(oldpath))) {
```

to prevent a case like,  :-)
- `newname` = `tflite.txt`
- `oldpath` = `a.tflite`

Reply:
@hyunsik-yoon 

`path.extname()` already returns the ext with '.' :-D
But it's a good idea to use endsWith instead of includes.

Reply:
@dayo09 

> path.extname() already returns the ext with '.' :-D

Oh. I didn't know that. :-) Thanks for letting me know.